CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+significant role in improving and maintaining the quality of our documentation.
+
+We use Docosaurus to build and deploy the documentation website. The Docs Page is hosted on Vercel, which supports automatic
+CI/CD deployments from the main branch. The website is available at [https://docs.papermc.io](https://docs.papermc.io).
+
+Using Docosaurus means that there are guides for how to write the documentation. You can find the Docosaurus documentation at
+[https://docusaurus.io/docs/category/guides](https://docusaurus.io/docs/category/guides). The documentation is written 
+in Markdown, which is a simple markup language that is easy to learn. Take a look at some of the existing documentation
+to get a feel for how it is written.
+
+### Adding a page
+
+To add a new page to the documentation, create a new Markdown file in the appropriate folder. For example, if you want to add
+a new page to the Paper Admin docs, create a new Markdown file in the `docs/paper/admin/` folder. The file name should be in 
+lowercase and use hyphens to separate words. For example, `my-new-page.md`.
+
+The page should start with a slug and title. For example:
+
+```markdown
+---
+slug: /reference/paper-plugins
+---
+
+# Paper Plugins
+
+This documentation page serves to explain all the new semantics and possible confusions that Paper plugins may introduce.
+```
+
+This will place the page at the `/paper/reference/paper-plugins` URL on the website. The slug should be unique and follow the   
+folder structure of the documentation. The title is the name of the page that will be displayed on the website and in the sidebar.
+
+You must also add the page to the sidebar. In this instance we will go to the `config/sidebar.paper.ts` file and
+add the file to the `items` tag for that section. You must use the file structure for that file. One Example would be 
+`admin/reference/paper-plugins` where you omit the file extension and use slashes to denote folders. Take a look at the 
+existing sidebar elements to get a feel for how it is structured.
+
+## Submitting Contributions
+
+If you wish to contribute to the PaperMC Docs, please follow these steps:
+
+1. Fork the repository to your GitHub account.
+
+2. Create a new branch from the `main` branch.
+
+3. Make your changes or add new documentation following the style guidelines mentioned below.
+
+4. Commit your changes and push them to your fork.
+
+5. Submit a pull request (PR) to the `main` branch of the Docs repository.
+
+6. Your PR will be reviewed, and feedback may be provided if necessary.
+
+7. Once your contribution meets the guidelines and requirements, it will be merged into the main repository.
+
+## Style Guidelines
+
+To maintain consistency and readability across the documentation, please adhere to the following style guidelines:
+
+1. **American English**: All documentation should be written in American English. Please use American spellings, grammar, and punctuation.
```

Comment:
Gotta be a website somewhere that covers the main differences for anyone who doesn't know all the quirks. Could be good to link that here

Reply:
idk if the site is any good, but they list the Our/or, ize/ise, yse/yze ae-oe/e, ence/ense, ogue/og spellings quite nice: https://www.oxfordinternationalenglish.com/differences-in-british-and-american-spelling/

Reply:
it’s got the word oxford in it so shrug